api/src/main/java/edu/wpi/first/shuffleboard/api/util/Debouncer.java

```diff
@@ -0,0 +1,61 @@
+package edu.wpi.first.shuffleboard.api.util;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * A simple class for debouncing method calls. Debouncing is used to prevent an expensive method from being called
+ * in rapid succession, only allowing it to run after a certain amount of time has passed without it being called
+ * again.
+ */
+public class Debouncer {
```

Comment:
I like that. I'll still call it `Debouncer` though, because it's still fundamentally debouncing a supplied function call